features/machine/machine.feature

```diff
@@ -141,4 +141,39 @@ Feature: Machine features testing
     Given I scale the machineset to +2
     Then the machineset should have expected number of running machines
 
+  # @author miyadav@redhat.com
+  # @case_id OCP-24363
+  @admin
+  @destructive
```

Comment:
do we need to be destructive? Seems like this scenario should not interfere with other scenarios.

Reply:
yes it will not interfere , we can remove destructive

Reply:
@miyadav it will interfere, because you scaled the cluster up and down, so this test can not run in parallel with others.

Reply:
thx @jhou1  added it and pushed

Reply:
Added destructive tag and validated - https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/82301/console

Reply:
@jhou1 , but only a special purpose machine set is scaled. Will this still interfere?

Reply:
@akostadinov yes, having a `destructive` tag makes sure no other tests run in parallel therefore no pods get schedule to the new node, because when we remove the machine in after scenario, the node will be deleted.

Reply:
Huh, good point.